pkg/adapter/config/kubernetes/sensitiveDoguConfigRepository.go

```diff
@@ -0,0 +1,45 @@
+package kubernetes
```

Comment:
🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-45 lines are duplicate of `pkg/adapter/config/kubernetes/doguConfigRepository.go:1-45`
<!-- __reviewdog__:ChBmOTg1YTNiZGUyNGI0YzMwEgpjaGVja3N0eWxl -->